src/openforms/scss/vendor/_django-two-factor-auth.scss

```diff
@@ -0,0 +1,6 @@
+body.login.centered-image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
not sure where these lines now suddenly come from, they are already in the 2fa stylesheet itself. The only line that's needed for this patch is the background color line.